install/pre-install-payload/scripts/reqs-deploy.sh

```diff
@@ -53,6 +54,24 @@ function install_vfio_gpu_containerd_artefacts() {
 	install_containerd_artefacts "vfio-gpu"
 }
 
+function install_nydus_snapshotter_artefacts() {
+	echo "Copying nydus-snapshotter artifacts onto host"
+
+	install -D -m 755 ${artifacts_dir}/opt/confidential-containers/bin/containerd-nydus-grpc /opt/confidential-containers/bin/containerd-nydus-grpc
```

Comment:
In the end, the /usr/local/bin/containerd-nydus-grpc -> /opt/confidential-containers/bin/containerd-nydus-grpc link not required, right?

Reply:
No, `containerd-nydus-grpc` doesn't have to be in `/usr/local/bin` as we call it directly to start the systemd service.